contracts/auction/LANDAuction.sol

```diff
@@ -188,6 +194,16 @@ contract LANDAuction is Ownable, LANDAuctionStorage {
         }
     }
 
+    /**
+    * @dev Set convertion fee rate
```

Comment:
(typo) convertion => conversion

Reply:
my bad, I have a lot of `convertion` here. I created [this issue](https://github.com/decentraland/land-auction/issues/22) in order to rename them. If I do it now I will have too much to rebase later with the other branches.

Reply:
👍